src/Components/ServiceScene/LogsJsonScene.tsx

```diff
@@ -567,6 +463,113 @@ export class LogsJsonScene extends SceneObjectBase<LogsJsonSceneState> {
     );
   };
 
+  private setStateFromUrl() {
```

Comment:
ugh ****ing whitespace

Reply:
I think `yarn lint:fix` causes damage, I think I ran it when I was trying to debug an error with eslint complaining about the yaml file updates. 

I'd vote for removing it and replacing with whatever we run in the precommit

Reply:
precommit is running lint-staged.config.js which runs prettier and eslint --fix which is very similar to what's in yarn lint:fix. I also install the prettier plugin in vsCode and it fixes everything on save. Does your IDE have that?